server/graphql/v2/mutation/CommentMutations.js

```diff
@@ -44,10 +45,18 @@ const commentMutations = {
         args.comment.ConversationId = idDecode(args.comment.ConversationId, IDENTIFIER_TYPES.CONVERSATION);
       }
 
+      if (args.comment.update) {
+        const update = await fetchUpdateWithReference(args.comment.update, req);
+        if (!update) {
+          throw new Error('This update does not exist');
+        }
```

Comment:
Thank to the code you've implemented we can simplify this as:

```suggestion
        const update = await fetchUpdateWithReference(args.comment.update, { loaders: req.loaders, throwIfMissing: true });
```

Reply:
@Betree : Thanks, I've committed this change. 👍🏽